src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -100,9 +100,6 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
     <DebugSymbols Condition="'$(DebugSymbols)'==''">false</DebugSymbols>
     <CheckForOverflowUnderflow Condition="'$(CheckForOverflowUnderflow)'==''">false</CheckForOverflowUnderflow>
-
-    <!-- Uncomment this once https://github.com/Microsoft/visualfsharp/issues/3207 gets fixed -->
-    <!-- <WarningsAsErrors Condition=" '$(WarningsAsErrors)' == '' ">NU1605</WarningsAsErrors> -->
```

Comment:
Please put this back as it was, but put the code as it should be in the comment. Otherwise, we're left without the line to replace the other two pending https://github.com/Microsoft/visualfsharp/issues/3207.